assets/js/script.js

```diff
@@ -1,42 +1,44 @@
-document.querySelector('#add-value').addEventListener('click', function() {
-    const itemName = document.querySelector('#item-name').value;
-    const itemPrice = document.querySelector('#item-price').value;
-
-    if (itemName && itemPrice) {
-      // Create a new item object
-      const item = {
-        name: itemName,
-        price: parseFloat(itemPrice)
-      };
-
-      // Retrieve existing items from localStorage
-      let items = JSON.parse(localStorage.getItem('items')) || [];
-
-      // Add new item to the list
-      items.push(item);
-      console.log(items);
-
-      // Save updated list to localStorage
-      localStorage.setItem('items', JSON.stringify(items));
-
-      // Clear input fields
-      document.querySelector('#item-name').value = '';
-      document.querySelector('#item-price').value = '';
-
-      // Update the item list UI
-      displayItems();
-    } else {
-      // Create <p> element to display the error message
-      const error = document.createElement('p');
-      error.id = 'error-message';
-      error.textContent = 'Error! Please Try Again.';
-
-      // Insert the error message below the button
-      document.querySelector('#add-value').insertAdjacentElement('afterend', error);
-      setTimeout(function () {
-        error.textContent = '';
-      }, 4000);
-    }
+document.querySelector('#add-value').addEventListener('click', function () {
+  const itemName = document.querySelector('#item-name').value;
+  const itemPrice = document.querySelector('#item-price').value;
+
+  if (itemName && itemPrice) {
+    // Create a new item object
+    const item = {
+      name: itemName,
+      price: parseFloat(itemPrice),
+    };
+
+    // Retrieve existing items from localStorage
+    let items = JSON.parse(localStorage.getItem('items')) || [];
+
+    // Add new item to the list
+    items.push(item);
+    console.log(items);
+
+    // Save updated list to localStorage
+    localStorage.setItem('items', JSON.stringify(items));
+
+    // Clear input fields
+    document.querySelector('#item-name').value = '';
+    document.querySelector('#item-price').value = '';
+
+    // Update the item list UI
+    displayItems();
+  } else {
+    // Create <p> element to display the error message
+    const error = document.createElement('p');
+    error.id = 'error-message';
+    error.textContent = 'Error! Please Try Again.';
+
+    // Insert the error message below the button
+    document
```

Comment:
Add value error message. Line 35
document.querySelector('#add-value').insertAdjacentElement('afterend', error);
on the same line should fix it all